__fixtures__/menus.json

```diff
@@ -1217,6 +1217,28 @@
       }
     ]
   },
+  "navbar-top-right": {
+    "label": "Navigation",
+    "items": [
+      { "label": "My Account", "url": "/myaccount", "submenu": null },
+      {
+        "label": "Subscribe",
+        "url": "/products?segmentId=f860e6c2-18af-ab30-cd5e-6e3a456f9265",
+        "submenu": null
+      }
+    ]
+  },
+  "navbar-top-right-anon": {
+    "label": "Navigation",
+    "items": [
+      { "label": "Sign In", "url": "/login?location=${currentPath}", "submenu": null },
+      {
+        "label": "Subscribe",
+        "url": "/products?segmentId=f860e6c2-18af-ab30-cd5e-6e3a456f9265",
+        "submenu": null
+      }
+    ]
+  },
```

Comment:
**Reviewer Notes:** This new data structure was added in a [origami navigation service PR](https://github.com/Financial-Times/origami-navigation-service/pull/922) and it can be seen in the output of the next navigation api https://next-navigation.ft.com/v2/menus